core/server/master/src/main/java/alluxio/master/file/PermissionChecker.java

```diff
@@ -282,9 +280,11 @@ private void checkInode(String user, List<String> groups, Inode<?> inode, Mode.B
     if (inode == null) {
       return;
     }
-    if (!mInodePermissionChecker.checkPermission(user, groups, inode, bits)) {
-      throw new AccessControlException(ExceptionMessage.PERMISSION_DENIED
-          .getMessage(toExceptionMessage(user, bits, path, inode)));
+    for (AclAction action : bits.toAclActions()) {
+      if (!inode.checkPermission(user, groups, action)) {
```

Comment:
Does `inode.checkPermission` need to pass for all of the `AclActions` or just one?